spec/features/current_organization_spec.rb

```diff
@@ -38,11 +38,7 @@
       end
     end
 
-    it "see organization that was selected before sign out" do
```

Comment:
@yunixon not sure how it fixes periodically failing spec if this spec was pending and wasn't run. If you fix pending specs then you fix it wrong. You change tested logic.